gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -883,6 +884,66 @@ def expand(self, pvalue):
     return pvalue.pipeline | Read(self._source)
 
 
+def _create_vcf_source(
+    file_pattern=None, compression_type=None, allow_malformed_records=None):
```

Comment:
It seems strange that any of these parameters would be optional since we supply them in the transform. This seems to be what was done in `ReadAllFromText` as well though...

Reply:
yeah, I noticed that too. It's the same in both other ReadAll* files. I assume it has something to do with using partial or the way the method is instantiated. I prefer to keep it like this for now to be consistent with other sources.

Reply:
Oh that's a good point. Since we're applying it with the second two parameters.